src/content/components/modal/usage.mdx

```diff
@@ -3,32 +3,43 @@ title: Modal
 tabs: ['Code', 'Usage', 'Style']
 ---
 
+### Modals communicate information via a secondary window and allow the user to maintain the context of a particular task.
+
 <AnchorLinks>
 
 - [General guidance](#general-guidance)
 - [Format](#format) 
 - [Variations](#variations) 
-- [Dismissal](#dismissal)
```

Comment:
Looks like this is no longer a stand alone section, and a sub section of **General guidance**. H2 index link should be removed.

Reply:
Thanks!